spec/features/hq/create_admin_users.rb

```diff
@@ -0,0 +1,17 @@
+require 'rails_helper'
+
+RSpec.feature 'Creating Admin Users', :type => :feature do
+
+  scenario 'After creation new Admin Users should be able to log in' do
+    i_sign_in_as_admin
+    visit new_hq_admin_user_path
+    fill_in 'Email', with: 'newadmin.user@example.com'
+    fill_in 'Password', with: 'Password'
+    fill_in 'Password confirmation', with: 'Password'
+    click_button 'Create Admin user'
```

Comment:
should we have a `i_create_an_admin_user_with_credentials(email, pass)` for these 4 lines too?